log4j/src/main/java/com/baeldung/log4j2/Log4j2RollingExample.java

```diff
@@ -1,7 +1,5 @@
 package com.baeldung.log4j2;
 
-import java.util.stream.IntStream;
-
 import org.apache.logging.log4j.LogManager;
```

Comment:
This file was already fixed in your other article - not sure why it would be showing up in this PR. You may want to merge the upstream/master branch from the main repo into your master branch, push to your fork, and then merge your master into the branch you're working with here.